tests/Spec.hs

```diff
@@ -53,6 +57,12 @@ main = Hspec.hspec $ do
       @(Fido2.PublicKeyCredential Fido2.AuthenticatorAssertionResponse)
       "tests/fixtures/login-complete"
       ignoreDecodedValue
+  describe "PublicKey" $ do
+    it "roundtrips" $ do
+      property $ \key -> do
+        let bs = CBOR.toLazyByteString (PublicKey.encodePublicKey key)
+        let rs = snd <$> CBOR.deserialiseFromBytes PublicKey.decodePublicKey bs
+        rs == pure key
```

Comment:
:+1: Good to have a roundtripping test. Did you check whether this test [covers](https://downloads.haskell.org/ghc/latest/docs/html/users_guide/profiling.html#hpc) all of the encoding and decoding logic?

Also, it would be good to fuzz the decoder too; right now we are only feeding it well-behaved inputs, not arbitrary bytestrings. We can let QuickCheck feed arbitrary byte strings of course, but that is not going to be effective because it is not coverage-driven. I don’t know how to fuzz a Haskell program with coverage-guided fuzzing, I think [AFL in QEMU mode](https://github.com/google/AFL/blob/master/qemu_mode/README.qemu) would work, but I haven’t tried it.

Reply:
I added coverage reports to CI; and yes it's covered